great_expectations/util.py

```diff
@@ -944,3 +944,17 @@ def generate_library_json_from_registered_expectations():
         library_json[expectation_name] = report_object
 
     return library_json
+
+
+def load_data_into_database(
+    table_name: str, csv_path: str, connection_string: str
+) -> None:
+    import pandas as pd
+    import sqlalchemy as sa
+
+    engine = sa.create_engine(connection_string)
+    engine.execute(f"DROP TABLE IF EXISTS {table_name}")
+    print(f"Dropping table {table_name}")
+    df = pd.read_csv(csv_path)
+    print(f"Creating table {table_name} from {csv_path}")
+    df.to_sql(name=table_name, con=engine, index=False)
```

Comment:
I don't think this belongs in the main `utils` file. It is only related to testing and we don't want  users inadvertently using this.